content/docs/12-enterprise-version/06-air-gap-repo.md

```diff
@@ -1,108 +1,730 @@
+
+  </InfoBox>
+
+   Assign the proper permissions and start the download script.
+  
+  <br />
+
+  ```shell
+  sudo chmod 755 ./airgap-edge-opensuse-rke.bin && sudo ./airgap-edge-opensuse-rke.bin
+  ```
+
+</Tabs.TabPane>
+
+<Tabs.TabPane tab="OpenSuse - Kubeadm" key="opensuse-kubeadm">
+
+   Download the binary by using the URL provided by the Palette support team. Change the version number as needed.
+
+  <br />
+
+  ```shell
+  curl --user XXXX:YYYYY https://<provided-by-support>/airgap/packs/3.3/airgap-edge-opensuse-kubeadm.bin \
+   --output airgap-edge-opensuse-kubeadm.bin
+  ```
+
+  <InfoBox>
+
+  If you receive a certificate error, use the `-k` or `--insecure` flag.
+
+  </InfoBox>
+
+   Assign the proper permissions and start the download script.
+  
+  <br />
+
+  ```shell
+  sudo chmod 755 ./airgap-edge-opensuse-kubeadm.bin && sudo ./airgap-edge-opensuse-kubeadm.bin
+  ```
+
+</Tabs.TabPane>
+
+
+</Tabs>
+
+<br />
+
+----
+
+<br />
+
+
+The next step of the installation process is to begin the deployment of an appliance using the instructions in the [Migrate Cluster to Enterprise Mode](/enterprise-version/deploying-an-enterprise-cluster). If you need to review the Spectro Cloud Repository details, issue the following command for detailed output.
+
+<br />
+
+```shell
+sudo /bin/airgap-setup.sh
+```
+
+<br />
 
 <InfoBox>
-Please make a note of the Palette Repository Details, Pack Registry Details, and Registry Certificate. These will be used while deploying the platform.
+
+You can review all the logs related to the setup of the private Spectro repository in  **/tmp/airgap-setup.log**.
+
 </InfoBox>
 
-* Download and execute a binary specific to the version of Palette you need to install. Contact the support team for version-specific binary. E.g.:
 
-`curl --user <user>:<password> https://repo.console.spectrocloud.com/airgap/packs/airgap-v1.14.24.bin -o airgap-v1.14.24.bin`
+# Validate
+
+You can validate that the Spectro Repository you deployed is available and ready for the next steps of the installation process. If you provided the appliance with an SSH key then you can skip to step five.
+
+1. Log in to vCenter Server by using the vSphere Client.
+
+
+2. Navigate to your Datacenter and locate your VM. Click on the VM to access its details page.
+
+
+3. Power on the VM.
+
+
+4. Click on **Launch Web Console** to access the terminal.
+
+
+5. Log in with the user `ubuntu` and the user password you specified during the installation. If you are using SSH, use the following command but ensure you specify the path to your SSH private key and replace the IP address with your appliance's static IP.
+
+  <br />
+
+  ```shell
+  ssh --identity_file ~/path/to/your/file ubuntu@10.1.1.1
+  ```
+
+
+6. Verify the registry server is up and available. Replace the `10.1.1.1` value with your appliance's IP address.
+
+  <br />
+
+  ```shell
+  curl --insecure https://10.1.1.1:5000/health
+  ```
+
+  Output:
```

Comment:
Do you want to name this 'Example Output' to match previous labels for example output?